aula1/divisao.php

```diff
@@ -0,0 +1,158 @@
+                        <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
+                    </div>
+                    <div class="offcanvas-body">
+                        <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                            <li class="nav-item">
+                                <a class="nav-link active" aria-current="page" href="index.php">Início</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#contato">Entre em contato</a>
+                            </li>
+                            <li class="nav-item dropdown">
+                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                                    Opções
+                                </a>
+                                <ul class="dropdown-menu">
+                                    <li><a class="dropdown-item" href="multiplicacao.php">&#x2716 Multiplicação</a></li>
+                                    <li><a class="dropdown-item" href="divisao.php">&#x2797 Divisão</a></li>
+                                    <li><a class="dropdown-item" href="soma.php">&#x2795 Soma</a></li>
+                                    <li><a class="dropdown-item" href="subtracao.php">&#x2796 Subtração</a></li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <main class="mb-5 text-center">
+        <section>
+            <img src="imagens/divisao.png" class="d-block w-100" alt="imagem divisao">
+        </section>
+
+        <section class="container">
+            <h1 class="mt-5">Tabuada da divisão</h1>
+            <p>Digite um número para ver a tabuada da divisão com ele.</p>
+        </section>
+
+        <section class="container">
+            <form action="" method="get">
+                <select name="tabuada">
+                    <?php
+                    for ($i = 1; $i <= $valorMaximoTabuada; $i++) {
+                        echo sprintf(
+                            '<option value="%d">%d</option>',
+                            $i,
+                            $i,
+                        );
+                    }
+                    ?>
+                </select>
+                <button type="submit">Ver</button>
+            </form>
+    
+            <div>
+                <?php
+                if ('' === $mensagemErro && null !== $tabuada && 0 !== $tabuada) {
+                    for ($j = 1; $j <= $valorMaximoTabuada; $j++) {
+                        $resultado = ($j * $tabuada) / $tabuada;
+    
+                        echo sprintf('<span>%d / %d = %d</span> </br>', ($j * $tabuada), $tabuada, $resultado);
```

Comment:
divisão ou multiplicação?
```suggestion
                        echo sprintf('<span>%d / %d = %d</span> </br>', ($j * $tabuada), $tabuada, $resultado);
```

Reply:
isso deu certo mesmo? acho que está renderizando o resultado 2x